assets/js/script.js

```diff
@@ -107,11 +132,24 @@ function calcTotalTime() {
     var activeIndex = parseInt($(".active").attr("index"))
     var a = JSON.parse(localStorage.getItem("objectClient"));
     // saves seconds in object
-    a[activeIndex].totalTime += seconds
+    a[activeIndex].totalTime += seconds;
     console.log(a[activeIndex].totalTime)
     console.log(a)
     localStorage.setItem("objectClient", JSON.stringify(a))
     secondsToTime();
+
```

Comment:
We have the populateTable() function in the table.js file, and the populateTable() function is called on Save

Do you want to put your table changes into that file instead?